index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300&display=swap" rel="stylesheet"> 
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div id="container">
+        <nav>
+            <div class="name">Peter Beshara</div>
+            <button type="button" class="menu"><img src="./media/Union.svg" alt="menu sign">
+            </button>
+        </nav>
+        <section class="section-1">
+            <header>Hey There. I’m Peter</header>
+            <h2>I am a Software Developer</h2>
+            <p>I can help you build a product , feature or website 
+                Look through some of my work and experience! If you like what you see and 
+                have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <div class="social-media">
+                <a href="https://github.com/Peter1907"><img src="./media/icons/Icon-GitHub.svg" alt="github icon"></a>
+                <a href="https://www.linkedin.com/in/peter-beshara-b33681241/"><img src="./media/icons/Icon-Linkedin.svg" alt="linkedin icon"></a>
+                <a href="https://angel.co/"><img src="./media/icons/Icon-Angelist.svg" alt="angelists icon"></a>
+                <a href="https://twitter.com/Peter_Beshara_"><img src="./media/icons/Icon-Twitter.svg" alt="twitter icon"></a>
+                <a href="https://medium.com/"><img src="./media/icons/Icon-Medium.svg" alt="medium icon"></a>
+            </div>
```

Comment:
- [ ] Great job with aligning the social media icons :clap: It would however been a better way to make then un-ordered list elements using the `<ul> .. <li><a href="#"></a></li> .. </ul>` format. This way your markup will be better organised and easier to read :sunglasses: >